### PR TITLE
Update tflite wheel version to 1.13.1

### PR DESCRIPTION
**Pull request by apivovarov**
This PR updates `tflite` wheel version to 1.13.1
I also added alternative way to get `tflite` module by cloning https://github.com/FrozenGene/tflite.git repo.

Merge it after web-data PR is merged https://github.com/dmlc/web-data/pull/190

### CODE REVIEW

**Comment by apivovarov** [comment]
@FrozenGene could you review it?

File: tutorials/frontend/from_tflite.py [hunk 1]
```diff
@@ -29,20 +29,32 @@
...
+    pip3 install tflite-1.13.1-py3-none-any.whl --user
 
     # For python2:
-    wget https://raw.githubusercontent.com/dmlc/web-data/master/tensorflow/tflite/whl/tflite-0.0.1-py2-none-any.whl
-    pip install tflite-0.0.1-py2-none-any.whl --user
+    wget https://raw.githubusercontent.com/dmlc/web-data/master/tensorflow/tflite/whl/tflite-1.13.1-py2-none-any.whl
+    pip install tflite-1.13.1-py2-none-any.whl --user
+
+
+or you could clone FrozenGene/tflite repo which contains generated tflite python module
```
Comment (FrozenGene) [inline tutorials/frontend/from_tflite.py]: I think it is better to let users do it themselves. In my opinion, we should restrict the repo under the umbrella of dmlc.

Comment (apivovarov) [reply tutorials/frontend/from_tflite.py]: line 57 still shows the option where people can generate tflite themself.
Options are:
* install `tflite` using pip
* simply clone `tflite` repo
* generate `tflite` using `flatc`

Comment (FrozenGene) [reply tutorials/frontend/from_tflite.py]: Yes. I saw it. My previous meaning is we restrict the repo url all are under the `dmlc`, not non-official ( for example FrozenGene/tflite.). However, I couldn't make sure whether we should have this restriction. Need more input from @tqchen @srkreddy1238 @yzhliu @kevinthesun  One solution maybe is we upload these generated python files to dmlc/web-data. Or just let users do it themselves like line 57.

Comment (apivovarov) [reply tutorials/frontend/from_tflite.py]: ok, I removed cloning `tflite` repo option from the tutorial.

**Comment by tqchen** [comment]
Given that we are not maintaining TFLite. I recommend we keep all the docker builds as binary, give instructions to install tensorflow and host the wheel image somewhere else.

**Comment by apivovarov** [comment]
> nd host the wheel image somewhere else.

@tqchen  I can replace `# Setup tflite from schema` in `ubuntu_install_tflite.sh` with `pip3 install tflite-1.13.1-py3-none-any.whl` after you merge https://github.com/dmlc/web-data/pull/190 which adds `tflite-1.13.1-py3-none-any.whl` to 'web-data`

**Comment by tqchen** [comment]
Sorry that I am not being clear.  I think we should build from source in the docker script when possible. As the binary wheel distribution is not really official.

**Comment by apivovarov** [comment]
@tqchen currently we use `tflite-0.0.1-py3-none-any.whl` wheel only in the tutorial
https://docs.tvm.ai/tutorials/frontend/from_tflite.html
This particular wheel version 0.0.1 was generated from schema 1.12

This PR updates tflite wheel to version 1.13.1 - `tflite-1.13.1-py3-none-any.whl`

In the tutorial we offer two option on how users can install `tflite` package
1. prebuilt wheel:
2.  generate TFLite package using `flatc`

in the docker file we use Option #2 - `flatc`

If the options listed above are fine with you then can you merge PR https://github.com/dmlc/web-data/pull/190
PR 190 adds `tflite-1.13.1-py3-none-any.whl` to web-data repo

**Comment by apivovarov** [comment]
@FrozenGene What you think if you make new release in you repo `FrozenGene/tflite` (version 1.13.1)?
You can attach new`tflite` wheels as the release Assets.
In TVM tutorial we can use wheel URLs pointing to release Assets instead of web-data.

Assets URLs will be smth like:
https://github.com/FrozenGene/tflite/releases/download/v1.13.1/tflite-1.13.1-py2-none-any.whl
https://github.com/FrozenGene/tflite/releases/download/v1.13.1/tflite-1.13.1-py3-none-any.whl

Looks like PR https://github.com/dmlc/web-data/pull/190 will never be merged.

**Comment by tqchen** [comment]
Given that we do not maintain tflite, I personally think it is not a good idea to distribute tflite binaries as part of the project. We could certainly advertise ``` FrozenGene/tflite```, or even better, point instructions to install tflite from the original tf repo

**Comment by apivovarov** [comment]
@FrozenGene I created tflite Release 1.13.1 in my forked repo
https://github.com/apivovarov/tflite/releases/tag/v1.13.1
Assets:
https://github.com/apivovarov/tflite/releases/download/v1.13.1/tflite-1.13.1-py2-none-any.whl
https://github.com/apivovarov/tflite/releases/download/v1.13.1/tflite-1.13.1-py3-none-any.whl

Can you do the same in your repo?

**Comment by apivovarov** [comment]
Changed `web-data/master/tensorflow/tflite/whl` wheel links to `FrozenGene/tflite/releases/download` links

**Comment by FrozenGene** [comment]
@apivovarov I think you could make a PR on my tflite repo, then I merge it. If there is a need, I could create a release branch after merging your assets wheel files

**Comment by FrozenGene** [comment]
@apivovarov I have done it. You could do a double check whether it is you want.

**Comment by apivovarov** [comment]
Thank you Zhao! The release and assets look good!
@tqchen I updated TFLite tutorial to use wheel files from FrozenGene/tflite/releases/download.
This PR should be ok to merge now.

**Comment by tqchen** [comment]
Thanks @FrozenGene @apivovarov , this PR is now merged

**Pull request merged**